doctrine/doctrine-bundle/1.6/config/packages/doctrine.yaml

```diff
@@ -8,12 +8,8 @@ parameters:
 doctrine:
     dbal:
         # configure these for your database server
-        driver: 'pdo_mysql'
-        server_version: '5.7'
-        charset: utf8mb4
-        default_table_options:
-            charset: utf8mb4
-            collate: utf8mb4_unicode_ci
+        driver: 'pdo_pgsql'
```

Comment:
Is there any good reason to twitch to pgsql for default projects? Mysql is still the most used for simple projects 😕

Reply:
Because Postgres is better,  has more features (JSON queries, advanced pagination and many many more...) and is as easy to use than MySQL.

Reply:
And has better default values

Reply:
A PR already exists about this. #411

Reply:
@dunglas  Can you revert this change?

Reply:
@fabpot you mean switching back to MySQL?